### PR TITLE
Add permissions to motion_submitter actions. Update tests.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: openslides_backend/action/actions/motion_submitter/sort.py [hunk 1]
```diff
@@ -27,3 +32,12 @@ def get_updated_instances(self, action_data: ActionData) -> ActionData:
             filter_id=instance["motion_id"],
             filter_str="motion_id",
         )
+
+    def get_meeting_id(self, instance: Dict[str, Any]) -> int:
```
Comment (FinnStutzenstein) [inline openslides_backend/action/actions/motion_submitter/sort.py:36]: Doesn't the submitter have a `meeting_id`?

**Review by FinnStutzenstein — COMMENTED** [review]
OK, but please see https://github.com/OpenSlides/openslides-backend/issues/609

File: openslides_backend/action/actions/motion_submitter/sort.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/motion_submitter/sort.py:36]: A submitter has a meeting_id, what is the point?

Comment (FinnStutzenstein) [reply openslides_backend/action/actions/motion_submitter/sort.py:36]: nevermind

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Looks good except same notes as in #603 regarding the duplication of the models

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Please use the enums here (and in all other places where you used explicit string) as @FinnStutzenstein remarked in #603. I missed it in some already merged PRs, too, I guess, so please revisit these as well.

**Comment by FinnStutzenstein** [comment]
> Please use the enums here (and in all other places where you used explicit string)

Should be done. These are action names, not permission names

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**